lib/pages/reader-page.js

```diff
@@ -12,24 +12,34 @@ export default class ReaderPage extends BaseContainer {
 		}
 		super( driver, by.css( '.is-section-reader' ), visit, url );
 	}
+
 	siteOfLatestPost() {
-		return this.driver.findElement( by.css( 'article.reader__card .site__domain' ) ).getText();
+		return this.driver.findElement( by.css( '.reader-visit-link' ) ).getAttribute( 'href' ).then( ( href ) => {
+			return href.split( '/' )[2];
+		} );
 	}
+
 	likeLatestPost() {
-		return driverHelper.clickWhenClickable( this.driver, by.css( 'article.reader__card .like-button' ) );
+		return driverHelper.clickWhenClickable( this.driver, by.css( '.reader-post-card .like-button' ) );
 	}
+
 	latestPostIsLiked() {
-		return this.driver.findElement( by.css( 'article.reader__card .like-button' ) ).getAttribute( 'class' ).then( ( classNames ) => {
+		return this.driver.findElement( by.css( '.reader-post-card .like-button' ) ).getAttribute( 'class' ).then( ( classNames ) => {
 			return classNames.includes( 'is-liked' );
 		} );
 	}
+
 	latestPostTitle() {
-		return this.driver.findElement( by.css( 'article.reader__card .reader__post-title' ) ).getText();
+		return this.driver.findElement( by.css( '.reader-post-card__title' ) ).getText();
 	}
+
 	commentOnLatestPost( comment ) {
 		driverHelper.clickWhenClickable( this.driver, by.css( '.comment-button' ) );
 		driverHelper.setWhenSettable( this.driver, by.css( '.comments__form textarea' ), comment );
-		driverHelper.clickWhenClickable( this.driver, by.css( '.comments__form button' ) );
-		return this.driver.wait( until.elementLocated( by.css( '.comment__moderation' ) ), this.explicitWaitMS, 'Could not see the comment moderation message after submitting a new comment' );
+		return driverHelper.clickWhenClickable( this.driver, by.css( '.comments__form button' ) );
+	}
+
+	waitForModeratedCommentToAppear() {
+		return this.driver.wait( until.elementLocated( by.css( '.comments__comment-moderation' ) ), this.explicitWaitMS, 'Could not see the comment moderation message after submitting a new comment' );
```

Comment:
We have a driverHelper method which does something similar and also caters for elements which are located but not hidden on screen, see: `driverHelper.waitTillPresentAndDisplayed(`